### ISSUE TITLE
Fix setting cookie expiry date

### ISSUE THREAD

**Issue by chenry35** [robotframework/SeleniumLibrary#891]
Hey,

I saw you deleted the optionnal argument expiry date on the keyword : add cookie.

Is there any way to add this option again? I want to add an expiry date on my cookie. 

Thanks

**Comment by aaltat** [comment]
The `expiry` argument was there but as you can see from the code from below (from release 1.8.0), it was not used anywhere. 
```
    def add_cookie(self,name, value, path=None, domain=None, secure=None,
            expiry=None):
        """Adds a cookie to your current session.
        "name" and "value" are required, "path", "domain" and "secure" are
        optional"""
        new_cookie = {'name'    : name,
                      'value'   : value}
        if path: new_cookie['path'] = path
        if domain: new_cookie['domain'] = domain
        #secure should be True or False so check explicitly for None
        if not secure is None: new_cookie['secure'] = secure
        self._current_browser().add_cookie(new_cookie)
```
I happy to review a PR which adds the argument and a logic which actually sets the expiry for the cookie.

**Comment by pekkaklarck** [comment]
As @aaltat commented, expiry was removed by #847 because it wasn't used. It would be a lot better to fix it instead. PR #930 looks so promising that I assign this issue to SL 3.0 rc 1 scope.

**Comment by pekkaklarck** [comment]
This issue requires a way to get the current cookie expiry time. PR #930 proposed introducing new `Get Cookie Expiry` keyword, but I'd rather add a keyword for getting all cookie information and added separate issue #932 about that.

**Comment by aaltat** [comment]
Done in #930

**Comment by pekkaklarck** [comment]
Added keyword documentation lacks information what attributes the returned object actually contains as well as a note that this keyword was added in SL 3.0. See these and few not-so-important comments at https://github.com/robotframework/SeleniumLibrary/pull/930/files

**Comment by pekkaklarck** [comment]
Keyword documentation should still be enhanced:

- Link to DateTime would be nice.
- Should add examples.
- Should have a note that prior to SL 3.0 expiry didn't actually work.

**Comment by pekkaklarck** [comment]
Final issues have been fixed by 17a24ffc4bd87cd54ec30afa672003b51e30ee66.
